Makefile

```diff
@@ -1,11 +1,13 @@
 clear-cache:
 	rm -rf var
 
+php-cs-fixer: clear-cache
+	docker-compose run --rm -T php /usr/local/bin/php-cs-fixer fix --no-interaction --verbose --dry-run
+
 phpunit: clear-cache
 	docker-compose run --rm -T php /usr/local/bin/php /app/vendor/bin/phpunit
 
 phpstan: clear-cache
-	docker-compose run --rm -T php /usr/local/bin/php /app/vendor/bin/phpstan analyse
```

Comment:
Why delete phpstan?
phpfixer and phpstan are compatibles and works fine together

Reply:
Very strange. I think it was a mistake.